q2_sample_classifier/_format.py

```diff
@@ -120,7 +120,7 @@ def _validate_(self, level):
     PredictionsFormat)
 
 
-class ImportanceFormat(model.TextFileFormat):
+class MultiColumnNumericFormat(model.TextFileFormat):
```

Comment:
```suggestion
class _MultiColumnNumericFormat(model.TextFileFormat):
```

Let's mark this as "private", that way an endeavoring developer doesn't get too carried away with this format.

Reply:
👍

Reply:
thanks for breaking my tests, dude 😆